lib/schemas/common/mapper.js

```
@@ -52,30 +52,27 @@ const baseMapperSchema = [
 						start: { type: 'string' },
 						end: { type: 'string' }
 					},
+					allOf: [
+						{
+							if: {
+								anyOf: [
+									{ required: ['start'] },
+									{ required: ['end'] }
+								]
+							},
+							then: {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)